snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/Acceptability.java

```diff
@@ -31,17 +31,19 @@ public enum Acceptability {
 	/**
 	 * Description is acceptable in language or dialect
 	 */
-	ACCEPTABLE(Concepts.REFSET_DESCRIPTION_ACCEPTABILITY_ACCEPTABLE),
+	ACCEPTABLE(Concepts.REFSET_DESCRIPTION_ACCEPTABILITY_ACCEPTABLE, "Acceptable"),
 
 	/**
 	 * Description is preferred in language or dialect
 	 */
-	PREFERRED(Concepts.REFSET_DESCRIPTION_ACCEPTABILITY_PREFERRED);
+	PREFERRED(Concepts.REFSET_DESCRIPTION_ACCEPTABILITY_PREFERRED, "Preferred");
```

Comment:
While this is okay for `en` displays, it might not be okay for others in the future, so I expect some additional improvement here - in a future Snow Owl version - as well to fetch the label of the acceptability from the db instead of hardcoding it here.